### DISCUSSION TITLE
Napalm credentials in custom script

### DISCUSSION THREAD

**Discussion by bogdancordos** [netbox-community/netbox#7968] [Q&A] [answered]
Hi,

Is there a way to import and use the Napalm credentials in a custom script?
Or hide the credentials to be displayed in the source code of the custom script? This is more of a python question, I guess.

Thank you

**Comment by candlerb** [comment, marked as answer]
I think you should be able to import `netbox.settings` and use `netbox.settings.NAPALM_USERNAME` etc.

**Reply by bogdancordos** [reply]
That work perfect. Thank you
